app/models/campaign.rb

```diff
@@ -22,4 +22,31 @@ class Campaign < ActiveRecord::Base
 
   has_attached_file :logo, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/:style/missing.png"
   validates_attachment_content_type :logo, content_type: /\Aimage\/.*\Z/
+
+  scope :joined_by, ->(org_id) { 
+    joins(:organization_campaigns).
+    where(organization_campaigns: {organization_id: org_id}) 
+  }
+
+  # TODO: String based SQL is not portable - make this nicer
+  scope :not_joined_by, ->(org_id) {
+    Campaign.where(
+      "NOT EXISTS (
+        SELECT 'X' 
+          FROM organization_campaigns oc 
+          WHERE oc.campaign_id = campaigns.id
+                AND oc.organization_id = ?
+      )", org_id
+    )
+  }
```

Comment:
I tried a couple of permutations of `where.not`, but couldn't get it to work.  It needs a parameterized LEFT OUTER JOIN to finish it (which doesn't exist in Rails).  Also tried Arel, which I've used successfully in other projects to do complex SQL like this, but it wasn't cooperating.  So just left it.